tests/beman/optional/CMakeLists.txt

```diff
@@ -31,12 +26,12 @@ target_link_libraries(
     beman_optional_test
     PRIVATE beman_optional GTest::gtest GTest::gtest_main
 )
-
 # Issue #32: Re-enable ASAN run CI/clang-19.
 #
 # Note: clang-19 + gtest_discover_tests + Asan setup causes errors on some
 # platforms. Temporary switch to gtest_add_tests and skip some Asan checks.
 # Change also applied for CI flows.
+include(GoogleTest)
```

Comment:
I think it was here, and during some review I got feedback from multiple people to put all includes at the top of the file.

Reply:
I've seen it both ways. You can't factor out CML.txt into distinct components, though, so spreading out includes across the file tends to mean you accumulate includes with no idea which uses what. You get one CML.txt per directory so the more effective maintainable style on projects I've been on have been to keep related things together. 
CMake is not C++.